### PR TITLE
Feat/create wallet

### PR DESCRIPTION
**Pull request by JM066**
Added a create wallet page with some backup phrases

### CODE REVIEW

**Review by IslomJonMirzakbarov — APPROVED** [review]

File: apps/web/src/App.tsx [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
-import { ReactQueryDevtools } from 'react-query/devtools';
+// import { ReactQueryDevtools } from 'react-query/devtools';
```
Comment (otabeknos1rov) [inline apps/web/src/App.tsx:2]: Can we remove comment if it is not necessary?

File: apps/web/src/components/Tag/index.tsx [hunk 2]
```diff
@@ -1,16 +1,37 @@
...
 }
 
-function Tag({ children, round, className }: ITag) {
-  return <div className={classNames(styles.Tag, { [styles.round]: round }, className)}>{children}</div>;
+function Tag({ name, round, addItem, removeItem, length, className }: ITag) {
+  const [toggle, toggleSwitch] = useToggle(false);
+
+  const handleClick = () => {
+    if (!toggle) addItem(name);
+    if (toggle) removeItem(name);
```
Comment (otabeknos1rov) [inline apps/web/src/components/Tag/index.tsx:20]: How it comes to use if/else instead double if checking ?

File: apps/web/src/components/Tag/index.tsx [hunk 3]
```diff
@@ -1,16 +1,37 @@
...
+
+    if (length == 12 && toggle) {
+      toggleSwitch();
+    }
```
Comment (otabeknos1rov) [inline apps/web/src/components/Tag/index.tsx:24]: Can we simplify it?

File: apps/web/src/pages/Onboarding/CreateWallet/ConfirmCreate/index.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (backupPhrases) {
+      const randomPhrase = RandomWord({ exactly: 8 });
+      const phrases = backupPhrases.split(' ').concat(randomPhrase);
+      const suffledPhrases = suffleItems(phrases);
+      setPhraseArr(suffledPhrases);
+    } else {
+      alert('Get backup phrase first');
```
Comment (otabeknos1rov) [inline apps/web/src/pages/Onboarding/CreateWallet/ConfirmCreate/index.tsx:28]: How it comes to use instead alert  a toast component , could you please request design for it ?

Comment (JM066) [reply apps/web/src/pages/Onboarding/CreateWallet/ConfirmCreate/index.tsx:28]: Yeah, the design is not ready so I just put an alert for now.

File: apps/web/src/components/Tag/index.tsx (hunk 3 above)
Comment (JM066) [reply apps/web/src/components/Tag/index.tsx:24]: Ok, let me see what I can do. Thanks

File: apps/web/src/components/Tag/index.tsx (hunk 2 above)
Comment (JM066) [reply apps/web/src/components/Tag/index.tsx:20]: Of course! I don't know why I didn't see that. THanks

**Pull request merged**